rpc/grpc/grpc.go

```diff
@@ -86,12 +92,18 @@ func (g *GrpcEndpoint) Allow(ctx context.Context, req *pb.AllowRequest) (*pb.All
 			logging.Printf("Caught error %v", err)
 			rsp.Status = pb.AllowResponse_REJECTED_SERVER_ERROR
 		}
-	} else {
-		rsp.Status = pb.AllowResponse_OK
-		rsp.TokensGranted = req.TokensRequested
-		rsp.WaitMillis = wait.Nanoseconds() / int64(time.Millisecond)
+
+		// Fail open
```

Comment:
nit: I find this comment confusing since failing closed is what happens right after.